demo/run_demo

```diff
@@ -46,6 +50,38 @@ do
 			echo "For example, to create the directory and then set the permissions use: 'mkdir ../logs; chmod uga+rws ../logs'"
 			exit 1
 		fi
+		continue
```

Comment:
Adding continue here means that the argument gets stripped off, which I don't think is intended.

Reply:
Thanks - wasn't sure about that.

Reply:
Fixed

Reply:
Confirmed that --timing works.